docs/en/sql-reference/functions/array-functions.md

```diff
@@ -2116,6 +2116,41 @@ Result:
 └───────────────────────────────────────────────┘
 ```
 
+## arrayAUC
```

Comment:
Thinking about it, we should instead of adding a new function, we should add a new parameter `scale` with default value `true` (behind parameters `arr_scores` and `arr_labels`) to existing function `arrayAUC`.

The syntax (l. 2126) would be: `arrayAUC(arr_scores, arr_labels[, scale])`.